notes/themes/_deprecated.txt

```diff
@@ -1,3 +1,24 @@
+speculative habitat
+===================
+
+:: Description
+Speculative places (on and off Earth) that humans might one day call home.
```

Comment:
Instead of "one day" we should say "hypothetically" I think.

Reply:
Agreed, if it were an active theme. This is in the deprecated file though.